neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/dialect/impl/RemoteNeo4jTupleAssociationSnapshot.java

```diff
@@ -64,8 +64,6 @@ else if ( targetNode.containsKey( targetColumnName ) ) {
 				else {
 					// Ex: @ElementCollection List<Embedded> examples
 
-					targetColumnName = targetColumnName.substring( collectionRole.length()  + 1 );
-
```

Comment:
This comes a bit out of the blue. Do you have an explanation on why it suddenly breaks after the upgrade of ORM?